go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/openshift-knative/serverless-operator
 
-go 1.21.0
+go 1.22
 
 toolchain go1.22.5
```

Comment:
Could you pls drop `toolchain` while it's already changing `go.mod` go version? Thanks! :)

Reply:
Done